origami.json

```diff
@@ -1,9 +1,6 @@
 {
-	"description": "Provides lazy loading functionality for images, pictures, iframes, and more. It is powered by the Intersection Observer API to detect when the visibility of elements changes.",
-	"keywords": ["lazy", "loading", "images"],
-	"origamiType": "module",
-	"origamiCategory": "components",
```

Comment:
There is a proposal to remove `origamiCategory` but we haven't accepted it just yet -- https://github.com/Financial-Times/origami/pull/120

Reply:
It's now been accepted 👍